window.py

```diff
@@ -1,26 +1,34 @@
 # -*- coding: utf-8 -*-
-# Copyright (c) 2009-11 Walter Bender
-# Copyright (c) 2012 Ignacio Rodríguez
+# Copyright (C) 2009-11 Walter Bender
+# Copyright (C) 2012 Ignacio Rodríguez
+#
 # This program is free software; you can redistribute it and/or modify
 # it under the terms of the GNU General Public License as published by
 # the Free Software Foundation; either version 3 of the License, or
 # (at your option) any later version.
 #
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
 # You should have received a copy of the GNU General Public License
-# along with this library; if not, write to the Free Software
-# Foundation, 51 Franklin Street, Suite 500 Boston, MA 02110-1335 USA
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301  USA
+
+import gi
+gi.require_version('Gtk', '3.0')
 from gi.repository import Gtk, Gdk
-import pygtk
-pygtk.require('2.0')
+
 from gettext import gettext as _
 
 from sugar3.graphics import style
-GRID_CELL_SIZE = style.GRID_CELL_SIZE
 
 from grid import Grid
 from sprites import Sprites
 from math import sqrt
 
+GRID_CELL_SIZE = style.GRID_CELL_SIZE
```

Comment:
rather than move this variable, it may be best to get rid of it and change the reference later from GRID_CELL_SIZE to style.GRID_CELL_SIZE.